package.json

```diff
@@ -8,30 +8,37 @@
   },
   "main": "dist/js/rivet-add-on-boilerplate.js",
   "scripts": {
-    "start": "gulp",
-    "build": "gulp release"
+    "start": "gulp buildDocs && gulp",
+    "build": "npm run clean && gulp release",
+    "clean": "rm -rf dist"
   },
   "author": "",
+  "component": "MyComponent",
```

Comment:
This is nice! Let's make the key name here something a little less generic, and more specific to Rivet/add-ons. My fear is that it could be confusing that this option could be confused by developers as something standard or specific to npm. 

Maybe something like:

`addOnName`